app/device/altimeter.py

```diff
@@ -0,0 +1,73 @@
+from app.utils.i2c import I2C
+from time import sleep
+
+MPL3115A2_ADDRESS = 0x60
+STATUS = 0x00
+OUT_P_MSB = 0x01
+OUT_P_DELTA_CSB = 0x08
+OUT_P_DELTA_MSB = 0X07
+OUT_T_MSB = 0x04
+WHOAMI = 0x0C
+CTRL_REG1 = 0x26
+PT_DATA_CFG = 0x13
+BAR_IN_MSB = 0x14
+
+class Altimeter (object):
+
+    def __init__(self):
+
+        while(True):
+            try:
+                self.bus = I2C(1, MPL3115A2_ADDRESS)
+                #Set oversample rate to 128
+                current_setting = self.bus.read_byte(CTRL_REG1)
+                new_setting = 0xb8
+                self.bus.write_byte(MPL3115A2_ADDRESS, CTRL_REG1, new_setting)
+                break
+
+            except IOError:
+                print("Device not connected.")
+                sleep(3)
+
+        # Enable event flags
+        self.bus.write_byte(MPL3115A2_ADDRESS, PT_DATA_CFG, OUT_P_DELTA_MSB)
+
+        # Toggel One Shot
+        setting = self.bus.read_byte(MPL3115A2_ADDRESS, CTRL_REG1)
+        if (setting & 0x02) == 0:
+            self.bus.write_byte(MPL3115A2_ADDRESS, CTRL_REG1, (setting | 0x02))
+
+
+
+    def read(self):
+        raw_data = self.bus.read_block(MPL3115A2_ADDRESS, OUT_P_MSB, 3)
+        return Altimeter.parse_raw_data(raw_data)
```

Comment:
the `return Altimeter.parse_raw_data(raw_data)` should be in a try catch block the way we did it [here](https://github.com/uOstar/barista/blob/master/app/device/gps.py)